runtime/polkadot/src/lib.rs

```diff
@@ -96,7 +96,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	impl_name: create_runtime_str!("parity-polkadot"),
 	authoring_version: 0,
 	spec_version: 9090,
-	impl_version: 0,
+	impl_version: 1,
```

Comment:
```suggestion
	impl_version: 0,
```

Reply:
so this is actually required because of the changes on line 568 and 520

Reply:
the spec_version has been bumped already since the last release, so I don't so